sdk/servicebus/azure-servicebus/azure/servicebus/_common/utils.py

```diff
@@ -60,23 +62,23 @@ def utc_from_timestamp(timestamp):
 
 
 def utc_now():
-    return datetime.datetime.now(tz=TZ_UTC)
+    return datetime.datetime.now(utc())
```

Comment:
Even if we're leaning towards not switching the verbiage elsewhere to keep this as a  single-point-of-contact, should likely yank the UTC class above since ideally the goal of this is that we're not relying on tit any more?

Reply:
Here's an update.